pkgs/applications/misc/gnome-secrets/default.nix

```
@@ -45,21 +44,17 @@ python3Packages.buildPythonApplication rec {
     glib
     gdk-pixbuf
     libadwaita
+    python3Packages.libpwquality.dev # Use python-enabled libpwquality
```

```suggestion
    python3Packages.libpwquality # Use python-enabled libpwquality
```
If that is not chosen automatically we need to look at its outputs.

---

Since it's already a specific output path, it must not recursively check. I don't know anything about how multiple output derivations work with buildInputs though.

---

It should choose dev automatically. Please give it a try.

---

Doesn't work.

```nix
nix-repl> "${python3Packages.libpwquality.py}"                        
"/nix/store/2ylr1v59zp1k9z9vw1l9hq3xgjaslagc-libpwquality-1.4.4-py"

nix-repl> "${python3Packages.libpwquality.py.dev}"
"/nix/store/kamd22r74zs83jlsaszm34sk9d5n56f1-libpwquality-1.4.4-dev"

nix-repl> "${lib.getOutput "dev" python3Packages.libpwquality.py}"
"/nix/store/2ylr1v59zp1k9z9vw1l9hq3xgjaslagc-libpwquality-1.4.4-py"
```